l10n_eu_product_adr_dangerous_goods/models/utility_models.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class StorageClass(models.Model):
+    _name = "storage.class"
+    _description = "Storage class"
+
+    name = fields.Char(string="Name", required=True)
+
+
+class PackagingType(models.Model):
+    _name = "packaging.type"
+    _description = "Packaging"
+
+    name = fields.Char(string="Name", required=True)
```

Comment:
Note for later: could we leverage this addon https://github.com/OCA/product-attribute/blob/14.0/product_packaging_type/models/product_packaging_type.py to handle packaging types?